cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityWritePrivilegesTest.scala

```diff
@@ -14,23 +14,208 @@ class SecurityWritePrivilegesTest extends DocumentingTest with QueryStatisticsTe
       "GRANT ROLE regularUsers TO jake",
       "GRANT ACCESS ON DATABASE neo4j TO regularUsers"
     )
-    synopsis("This section explains how to use Cypher to manage write privileges for Neo4j.")
+    synopsis("This section explains how to use Cypher to manage write privileges on graphs.")
 
     p(
       """
+        |* <<administration-security-writes-create, The `CREATE` privilege>>
+        |* <<administration-security-writes-delete, The `DELETE` privilege>>
+        |* <<administration-security-writes-set-label, The `SET LABEL` privilege>>
+        |* <<administration-security-writes-remove-label, The `REMOVE LABEL` privilege>>
+        |* <<administration-security-writes-set-property, The `SET PROPERTY` privilege>>
+        |* <<administration-security-writes-merge, The `MERGE` privilege>>
         |* <<administration-security-writes-write, The `WRITE` privilege>>
+        |* <<administration-security-writes-all, `ALL GRAPH PRIVILEGES`>>
         |""".stripMargin)
 
         p(
       """
-        |There are several separate write privileges:
+        |Write privileges are defined for different parts of the graph:
         |
-        |* `WRITE` - this privilege can only be assigned to all nodes, relationships, and properties in the entire graph.
+        |* `CREATE` - allows creating nodes and relationships
+        |* `DELETE` - allows deleting nodes and relationships
+        |* `SET LABEL labels` - allows setting the specified node labels using the `SET` clause.
+        |* `REMOVE LABEL labels` - allows removing the specified node labels using the `REMOVE` clause.
+        |* `SET PROPERTY +{props}+` - allows setting properties on nodes and relationships.
+        |
+        |There are also compound privileges which combine the above specific privileges:
+        |
+        |* `MERGE` - allows match, create and set property to permit the `MERGE` command.
+        |* `WRITE` - allows all write operations on an entire graph.
+        |* `ALL GRAPH PRIVILEGES` - allows all read and write operation on an entire graph.
```

Comment:
I think in each of these bullets, "allows" can be replaced with "enables".
Also, some bullets are missing full stops.

If it's helpful, I don't mind pushing editorial changes directly to the PR - ping me if you'd prefer that.

Reply:
When thinking about the difference between 'allows' and 'enables' I cannot help but imagine that is something is 'not allowed' that is an error, while if something is 'not enabled' it just means nothing will happen. Now when we compare READ and WRITE privileges, the READ privileges will simply control what you soo (with no error) and so better match the word _enables_, while WRITE privileges can result in actual errors (you get an exception if you try write something you are not allowed to), which certainly better matches the word _allows_. Would it make sense to make use of _BOTH_ words then, but with consistent association with these two very different cases? This will also help maintain clarity for the user that READ and WRITE privileges behave in such a very different way.

Reply:
thanks @craigtaverner for the clarity - I'm in agreement, let's use both where appropriate.

Additionally, when we finally get to building a documentation styleguide, these descriptions of when to use "enables" and "allows" are very clear! Many thanks!

Reply:
I like that distinction as well @craigtaverner. We should probably go through the docs later though and see that it is consistent everywhere since we have not written them with this in mind.

Reply:
I've made a commit (will push soon) that defines _enable_ and _allow_ in the previous section (fine-grained) and then uses _enable_ in the reads section and _allow_ in the writes section.